master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -668,11 +671,17 @@ private[celeborn] class Master(
         fetchPort,
         replicatePort,
         disks.map { disk => disk.mountPoint -> disk }.toMap.asJava,
-        userResourceConsumption,
         System.currentTimeMillis(),
         highWorkload,
         workerStatus,
         requestId)
+      statusSystem.updateWorkerResourceConsumptions(
```

Comment:
Can we also use this way to update resource consumption when worker register to master